src/azure-cli/setup.py

```diff
@@ -137,6 +137,7 @@
     'jsmin~=2.2.2',
     'jsondiff==1.2.0',
     'packaging~=20.9',
+    'PyGithub==1.38',
```

Comment:
`github` module is not imported anywhere. Is this dependency necessary?

Reply:
I should have added it to this PR instead, it will be used in this one https://github.com/Azure/azure-cli/pull/18261. And yes we shouldn't pin, we can use ~= and unpin. Thanks for bringing this up

Reply:
Could you change the dependency to use `~=` instead?